File: fabric/src/main/java/client/TemplateModFabricClient.java
package com.example.template.fabric.client;

import com.example.template.client.TemplateModCommonClient;
import net.fabricmc.api.ClientModInitializer;

public class TemplateModFabricClient implements ClientModInitializer {
	@Override
	public void onInitializeClient() {
		TemplateModCommonClient.init();
	}
}

File: fabric/src/main/kotlin/client/TemplateModFabricClient.kt
package com.example.template.fabric.client

import com.example.template.client.TemplateModCommonClient
import net.fabricmc.api.ClientModInitializer

class TemplateModFabricClient : ClientModInitializer {
	override fun onInitializeClient() {
		TemplateModCommonClient
	}
}
